examples/case_params_with_units.py

```diff
@@ -53,13 +53,14 @@
 
 del params_as_dict["fluid_properties"]
 
-errors, warnings = validate_flow360_params_model(params_as_dict)
-pprint(errors)
+with fl.SI_unit_system:
+    errors, warnings = validate_flow360_params_model(params_as_dict)
+    pprint(errors)
```

Comment:
validation should not be called inside context

Reply:
This fails without the context - in case-editor branch we have the unit system context passed as an argument, but on develop this is the only way to prevent the "Params must be created within a unit system context (...)" error message. Once changes from case-editor are applied this can be discarded.